CommonTools/RecoAlgos/python/TOFPIDProducer_cfi.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+TOFPIDProducer = cms.EDProducer( "TOFPIDProducer",
```

Comment:
the instance better follow the lower case convention -> ```tofPIDProducer```.
Perhaps even ```tofPID``` is good enough: we typically drop the "Producer" from the collection names